variables.tf

```diff
@@ -335,6 +335,24 @@ variable "key_vault_secrets_provider_enabled" {
   nullable    = false
 }
 
+variable "kms_enabled" {
+  type        = bool
+  description = "(Optional) Enable Azure KeyVault Key Management Service."
+  default     = false
```

Comment:
Would an extra `nullable = false` here be better?

Reply:
I can do this, sure. 👍

Reply:
Done